src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx

```diff
@@ -117,6 +128,7 @@ const FlaggingSystemModal = ({
                 base: 'full',
                 md: 'initial',
               }}
+              disabled={buttonIsDisabaled}
```

Comment:
@Adebesin-Cell instead this can be done in a single line
```suggestion
              disabled={flagContent.trim().length === 0}
```

Reply:
Okay thanks @Royal-lobster.